browser-test/src/trusted_intermediary.test.ts

```diff
@@ -961,13 +964,56 @@ test.describe('Trusted intermediaries', {tag: ['@uses-fixtures']}, () => {
       await tiDashboard.gotoTIDashboardPage(page)
       await waitForPageJsLoad(page)
 
+      await tiDashboard.goToAccountSettingsPage(page)
+      await waitForPageJsLoad(page)
+
       await validateScreenshot(
         page.getByTestId('org-members-table'),
         'org-members-table-many',
       )
     })
   })
 
+  test.describe('ti navigation', () => {
+    test('marks the correct tab as current and matches screenshot', async ({
+      page,
+      tiDashboard,
+    }) => {
+      await loginAsTrustedIntermediary(page)
+      await tiDashboard.gotoTIDashboardPage(page)
```

Comment:
suggestion (non-blocking): use `test.step` to break these up into logical chunks.